oi/type_graph/ClangTypeParser.cpp

```diff
@@ -385,6 +388,11 @@ Type& ClangTypeParser::enumeratePointer(const clang::PointerType& ty) {
   return makeType<Reference>(ty, t);
 }
 
+Type& ClangTypeParser::enumerateMemberPointer(
+    const clang::MemberPointerType& ty) {
+  return makeType<Primitive>(ty, Primitive::Kind::StubbedPointer);
```

Comment:
Thinking about it some more, wouldn't we want to follow a MemberPointer with the same rules as for regular pointers? i.e. never follow function pointers and otherwise check the `chasePointer` value.

I think the case we encountered was for a function pointer(?) so not following it makes sense for now.

Maybe just leave a TODO comment in case we ever come back to this.

Reply:
You're probably right. I'll put a TODO in as suggested but leave it as it is for now. You're right - the case that was failing was a function pointer.

Reply:
Thanks as always @JakeHillion for the insight!